playbook.yml

```diff
@@ -154,6 +154,13 @@
       - yum update -y
       - yum install -y /tmp/install/tableau-tabcmd-gold-master.noarch.rpm
 
+  - name: Install zip & unzip - to extract Tableau files
```

Comment:
Can this be used instead of `shell`: https://docs.ansible.com/ansible/latest/modules/yum_module.html
Also it may worth grouping all yum packages under one module and install all at the beginning of the build.

Reply:
Finally, I've implemented that fairly trivial change.